### PR TITLE
Playground: update code when changing theme

### PR DESCRIPTION
**Pull request by ourystd**
Fixing the issue #34

### CODE REVIEW

**Comment by ourystd** [comment]
@fkhadra could you check this please. I noticed this issue on the doc and wanted to something.

I don't have big experience in open sourcing yet :) 
Thanks !

File: src/components/ContainerCode.tsx [hunk 1]
```diff
@@ -33,7 +33,12 @@ export const ContainerCode: React.FC<ContainerCodeProps> = ({
   pauseOnFocusLoss,
   isDefaultProps,
   draggable,
-}) => (
+  theme = "light",
+}) => {
+  console.log({theme});
```
Comment (fkhadra) [inline src/components/ContainerCode.tsx:38]: Hey sorry for the late reply. You can drop the console.log

Comment (ourystd) [reply src/components/ContainerCode.tsx]: Oh, no problem! Sorry I mine was late too  :) 

I've removed the the console.log. You can check out.
Once again, thanks for the work. Your package is really great!

**Comment by relaxxpls** [comment]
@fkhadra could you merge this? It'll help a ton. Thanks for react-toastify though!

**Comment by fkhadra** [comment]
Thanks for the contribution 🙏 and sorry for being late

**Comment by ourystd** [comment]
My pleasure !

**Pull request merged**